.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: java
+dist: trusty
+sudo: true
+before_cache:
+- rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+cache:
+  directories:
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
+  - $HOME/.m2
+jdk:
+- oraclejdk8
+script: ./gradlew jacocoTestReport ;
+env:
+- ARTIFACTORY_USERNAME=gatkci
+- secure: E0LWXgX3aWSE/DWHXXDx4vrAq4uX6vKg402wToaZ5otbHQ/UP0H7/FA5jomavAXoC46oMVHZcEltZ5OVhuJ0NW8yYxUCecJ1D/YvVQmnfFABcV/qLM+k4e2rYQOKVw/pejB2gG8XdTA+XE2WyTeENbmIkputS8f1ndKWCmZxuuk=
```

Comment:
I'd add a comment saying what this secure variable is for. I've learned from experience that it can be confusing if we get multiple of them or if the yml gets complicated.